### ISSUE TITLE
New text-index command is not showing results in search box

### ISSUE THREAD

**Issue by duncancarroll** [GMOD/jbrowse-components#2277]
Hi folks! 👋

First off, thanks for the great components & tools!

My issue is, I have a Jbrowse 1.3.5 React component running [here](https://smm-data.herokuapp.com/dna) which shows a GRCh37 assembly with a Variant Track of my SNPs originally from 23AndMe.  It seems to work great, however I've run into problems trying to make it searchable.

For some reason I couldn't find  `jbrowse text-index` as a command in 1.3.5 (this seemed strange since I thought it was an existing feature), so I pulled latest `main` and built the CLI.  I was then able to run `jbrowse text-index` which produced the various trix files, about 50Mb which seems to list all my variants by `rs________` Reference SNP.  I pushed these to S3 where they are [world-readable](https://bucketeer-b2ec4285-3f6c-49c8-9753-fe99a1df2ac6.s3.amazonaws.com/public/raw/dna/trix/GRCh37.ix) and added them into my config file per the [instructions](https://jbrowse.org/jb2/docs/config_guide#text-searching).

The component compiles fine, however nothing seems to show up in the search box.  It's possible I'm misunderstanding something but I was expecting to be able to type in something like `rs4747667` and jump to that variant's location.  Is it possible the new text-index isn't backwards-compat with 1.3.5?  Any help is appreciated.  Thanks!

**To Reproduce**
A copy of the config file I'm using is [here](https://bucketeer-b2ec4285-3f6c-49c8-9753-fe99a1df2ac6.s3.amazonaws.com/public/raw/dna/config.json) which contains public URLs to all relevant files.  This is passed to a React component via the normal means:
```
const textSearchConfig = {
    assembly,
    aggregateTextSearchAdapters,
    tracks,
    defaultSession,
    location
  }

    const state = createViewState(textSearchConfig);
    return <JBrowseLinearGenomeView viewState={state} />;
```
**Expected behavior**
Entering a Reference SNP code e.g. `rs4747667` would jump to that location.

**Version:**
1.3.5 of React component with latest `main` of JBrowse2 CLI tools.

Browser is latest Chrome

**Comment by cmdcolin** [comment]
@duncancarroll this is great to see the enthusiasm for this. the text-indexing actually just got merged into the main line branch on friday and it hasn't been formally released yet! Looks like it accidentally got deployed to the website...we have a regex that deploys commits with [update docs] to the website but it wasn't supposed to run yet hehe

If you want to give it a shot, and you have already if you also made a main branch build, and used the main branch CLI, you could probably get it working :)

Basically

```
git clone repo
cd jbrowse-components
yarn # install deps
cd products/jbrowse-react-linear-genome-view
yarn build
```

Then use this as a yarn link'd entity in your project or just rm -rf node_modules/@jbrowse/react-linear-genome-view and replace it with the products/jbrowse-react-linear-genome-view/dist folder

**Comment by duncancarroll** [comment]
Oh awesome!  I'll give it a spin and report back---thanks!

**Comment by duncancarroll** [comment]
For some odd reason I'm getting Typescript compile errors when I run `yarn build` at the `tsdx build` step:

```
jbrowse-react-linear-genome-view/src/createModel/createSessionModel.ts(69,32): semantic error TS2749: 'TextSearchManager' refers to a value, but is being used as a type here. Did you mean 'typeof TextSearchManager'?
```

It's not urgent & I'm happy to just wait for the next release so if this is a complicated fix by all means feel free to close this issue.
